pkg/policydef/policydef.go

```diff
@@ -68,4 +69,8 @@ type Policy interface {
 	// GetAction must return the configured action from the policy's config. No
 	// validation is needed by the policy, it will be done centrally.
 	GetAction(ctx context.Context, c *github.Client, owner, repo string) string
+
+	// GetOrgActionConfig must return the configuration for the action from org and repo config.
+	// The return value must follow configdef.OrgActionConfig.
+	GetOrgActionConfig(ctx context.Context, c *github.Client, owner, repo string) configdef.OrgActionConfig
```

Comment:
Looks policydef should not have dependency on config package. I made configdef to communicate `enforce <=> policy` and `enforce <=> issue`.
I'm not sure this approach is good or not honestly... Could you look into it 🙇 ?